api/github_service/server.js

```diff
@@ -0,0 +1,14 @@
+const { SceHttpServer } = require('../util/SceHttpServer');
+
+function main() {
+  const API_ENDPOINTS = [
```

Comment:
you can also just do
```suggestion
  const API_ENDPOINTS = '/routes/`
```
and it should work the same